app/views/govuk_publishing_components/audit/_component_contents.html.erb

```diff
@@ -0,0 +1,82 @@
+<%
+  total_components = components.length
+  components_with_stylesheet = 0
+  components_with_print_stylesheet = 0
+  components_with_javascript = 0
+  components_with_test = 0
+  components_with_javascript_test = 0
+  components_with_helper = 0
+
+  components.each do |component|
+    components_with_stylesheet += 1 if component[:stylesheet]
+    components_with_print_stylesheet += 1 if component[:print_stylesheet]
+    components_with_javascript += 1 if component[:javascript]
+    components_with_test += 1 if component[:tests]
+    components_with_javascript_test += 1 if component[:js_tests]
+    components_with_helper += 1 if component[:helper]
+  end
+%>
+<table class="govuk-table">
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th scope="col" class="govuk-table__header sticky-table-header">Component (<%= total_components %>)</th>
+      <th scope="col" class="govuk-table__header sticky-table-header">CSS (<%= components_with_stylesheet %>)</th>
+      <th scope="col" class="govuk-table__header sticky-table-header">Print CSS (<%= components_with_print_stylesheet %>)</th>
+      <th scope="col" class="govuk-table__header sticky-table-header">JS (<%= components_with_javascript %>)</th>
+      <th scope="col" class="govuk-table__header sticky-table-header">Test (<%= components_with_test %>)</th>
+      <th scope="col" class="govuk-table__header sticky-table-header">JS test (<%= components_with_javascript_test %>)</th>
+      <th scope="col" class="govuk-table__header sticky-table-header">Helper (<%= components_with_helper %>)</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <% components.each do |component| %>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">
+          <a href="<%= component[:link] %>" class="govuk-link"><%= component[:name] %></a>
+        </th>
+        <td class="govuk-table__cell">
+          <% if component[:stylesheet] %>
+            <strong class="govuk-tag govuk-tag--green">
+              <a href="<%= component[:stylesheet] %>" class="govuk-link">Yes</a>
+            </strong>
+          <% end %>
```

Comment:
If `component[:stylesheet]` isn't present, we're left with an empty table cell which isn't ideal. Could there be a non-strong "No" present?
```suggestion
          <% if component[:stylesheet] %>
            <strong class="govuk-tag govuk-tag--green">
              <a href="<%= component[:stylesheet] %>" class="govuk-link">Yes</a>
            </strong>
          <% else %>
            No
          <% end %>
```

Reply:
That's a fair point, but there's a lot of information in this table and it's currently easier to compare 'yes' with empty than compare 'yes' with 'no'. Thanks for making this non-blocking, maybe if we can get the auditing to a better position it might be good to do a review of the pages and include this question in it.